sftpgo/terraform/gcp/variables.tf

```diff
@@ -0,0 +1,21 @@
+variable "namespace" {
+  type = string
+  default = "sftpgo"
+}
+
+variable "cluster_name" {
+  type = string
+}
+
+variable "project_id" {
+  type = string
+  description = <<EOF
+The ID of the project in which the resources belong.
+EOF
+}
+
+variable "roles" {
+  type        = list(string)
+  description = "A list of roles to be added to the sftpgo workload identity service account"
+  default     = []
```

Comment:
should you add a default here?

Reply:
No default value is required initially. Only needed for the GCS output destination. 
Should it be added even if it is not used or let users assign it?